### PR TITLE
Implement the likeComment in UserActions+ObjectProvider

### PR DESCRIPTION
**Pull request by Tbruni85**
## JIRA
https://jira.superawesome.tv/browse/KWS-2205

## DESCRIPTION
`LikeComment` service provided by *Protobufs* implemented and tested in the *Data SDK*

## SCREENSHOTS
n/a

### CODE REVIEW

File: Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift [hunk 1]
```diff
@@ -318,4 +318,255 @@ class UserActionsObjectProviderTests: XCTestCase {
             })
         }
     }
+    
+    // MARK: Like comment
+    func testUserActionLikeComment_SuccessReasponse() {
```
Comment (GuilhermeMota93) [inline Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift]: Typo in 'Response'

File: Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift [hunk 2]
```diff
@@ -318,4 +318,255 @@ class UserActionsObjectProviderTests: XCTestCase {
...
+                                               token: userToken,
+                                               completionHandler: { (error) in
+                                                    
+                expect(error).to(beNil())
+                done()
+            })
+        }
+    }
+    
+    func testUserActionLikeComment_FeedNotFoundReasponse() {
```
Comment (GuilhermeMota93) [inline Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift]: Same typo in 'Response'

File: Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift [hunk 3]
```diff
@@ -318,4 +318,255 @@ class UserActionsObjectProviderTests: XCTestCase {
...
+                expect((error as! ErrorResponse).usernameAlreadyTaken).to(beFalse())
+                expect((error as! ErrorResponse).usernameForbidden).to(beFalse())
+                expect((error as! ErrorResponse).message).to(equal("child \"feedItemId\" fails because [\"feedItemId\" must be a valid GUID]"))
+                expect((error as! ErrorResponse).validationError).to(beTrue())
+                done()
+            })
+        }
+    }
+    
+    func testUserActionLikeComment_CommentNotFoundReasponse() {
```
Comment (GuilhermeMota93) [inline Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift]: Same typo in 'Response'

File: Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift [hunk 4]
```diff
@@ -318,4 +318,255 @@ class UserActionsObjectProviderTests: XCTestCase {
...
+                                                expect((error as! ErrorResponse).usernameAlreadyTaken).to(beFalse())
+                                                expect((error as! ErrorResponse).usernameForbidden).to(beFalse())
+                                                expect((error as! ErrorResponse).message).to(equal("child \"commentId\" fails because [\"commentId\" must be a valid GUID]"))
+                                                expect((error as! ErrorResponse).validationError).to(beTrue())
+                                                done()
+            })
+        }
+    }
+
+    func testUserActionLikeComment_FailReasponse() {
```
Comment (GuilhermeMota93) [inline Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift]: Same typo in 'Response'

File: Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift [hunk 5]
```diff
@@ -318,4 +318,255 @@ class UserActionsObjectProviderTests: XCTestCase {
...
+                                               token: userToken,
+                                               completionHandler: { (error) in
+
+                expect(error).toNot(beNil())
+                done()
+            })
+        }
+    }
+
+    func testUserActionDislikeComment_SuccessReasponse() {
```
Comment (GuilhermeMota93) [inline Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift]: Same typo in 'Response'

File: Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift [hunk 6]
```diff
@@ -318,4 +318,255 @@ class UserActionsObjectProviderTests: XCTestCase {
...
+                                               token: userToken,
+                                               completionHandler: { (error) in
+                                                
+                expect(error).to(beNil())
+                done()
+            })
+        }
+    }
+
+    func testUserActionDislikeComment_FailReasponse() {
```
Comment (GuilhermeMota93) [inline Example/PopJamSDK_Tests/UserActions/UserActions+ObjectProviderTests.swift]: Same typo in 'Response'

**Comment by devgabrielcoman** [comment]
Seems to be 👍

**Pull request merged**